lib/src/reboot.rs

```diff
@@ -14,7 +14,7 @@ pub(crate) fn reboot() -> anyhow::Result<()> {
     let _ = std::io::stdout().flush();
     let _ = std::io::stderr().flush();
     Task::new("Rebooting system", "systemd-run")
-        .args(["--message=Initiated by bootc", "reboot"])
+        .args(["reboot", "--", "--message=Initiated by bootc"])
```

Comment:
As `reboot --help` says

> This is a compatibility interface, please use the more powerful 'systemctl reboot' command instead.

And to this point 

```
+ reboot --message foo
reboot: unrecognized option '--message'
```

So I don't see how this can work.

What we want is:

`systemd-run systemctl reboot --message=Initiated by bootc`